platform/javascript/javascript_main.cpp

```diff
@@ -32,13 +32,20 @@
 #include "main/main.h"
 #include "platform/javascript/display_server_javascript.h"
 #include "platform/javascript/os_javascript.h"
+#include "scene/main/node.h"
 
 #include <emscripten/emscripten.h>
 #include <stdlib.h>
 
 static OS_JavaScript *os = nullptr;
 static uint64_t target_ticks = 0;
 
+extern "C" EMSCRIPTEN_KEEPALIVE void _request_quit_callback(char *p_filev[], int p_filec) {
+	if (os && os->get_main_loop()) {
+		os->get_main_loop()->notification(Node::NOTIFICATION_WM_CLOSE_REQUEST);
```

Comment:
There seems to be `DisplayServer::WINDOW_EVENT_CLOSE_REQUEST`, which is what triggers emitting `NOTIFICATION_WM_CLOSE_REQUEST` in `Windows`. Maybe this is better suited here than emitting a Node notification?